src/bot/commands/subscribe.rs

```diff
@@ -189,6 +189,15 @@ mod subscribe_tests {
             .create();
         let feed_url = format!("{}{}", mockito::server_url(), path);
 
+        let response_string = "{\"ok\":true,\"result\":{\"message_id\":2746,\"from\":{\"id\":1276618370,\"is_bot\":true,\"first_name\":\"test_el_bot\",\"username\":\"el_mon_test_bot\"},\"date\":1618207352,\"chat\":{\"id\":275808073,\"type\":\"private\",\"username\":\"Ayrat555\",\"first_name\":\"Ayrat\",\"last_name\":\"Badykov\"},\"text\":\"Hello!\"}}";
```

Comment:
maybe let's create a function to move this logic duplicated in three tests?

Reply:
[Deliver Mockito server func](https://github.com/ayrat555/el_monitorro/pull/252/commits/9fb116072aaec67027a058b7452a98820444be97)